loaders/parser.go

```diff
@@ -47,16 +47,16 @@ func NewSpannerLoaderFromDDL(fpath string) (*SpannerLoaderFromDDL, error) {
 	for _, ddl := range ddls {
 		switch val := ddl.(type) {
 		case *ast.CreateTable:
-			v := tables[val.Name.Name]
+			v := tables[val.Name.SQL()]
```

Comment:
I want to align the writing style with v2.
https://github.com/cloudspannerecosystem/yo/blob/44c1d636a3cb215cbf4e45893496770b536b032e/v2/loader/schema_parser_source.go#L61-L64

Reply:
Thank you! I'll try it